src/subcommand/torrent/announce.rs

```diff
@@ -0,0 +1,135 @@
+use crate::common::*;
+
+const URI_HELP: &str = "Announce an infohash and list the response (a compact peer list).";
```

Comment:
I think this should be more direct, like "Print torrent peers". Also, it should be named after the name of the command, so "PEERS_HELP". (Pending command name.)

Reply:
Actually, if the command name is `announce` I think "Announce a torrent and print the returned peers." is better.